regex-assembly/exclude/unix-shell-fps-pl1.ra

```diff
@@ -197,7 +200,11 @@ cpulimit
 crash
 crash@
 crash~
+##! Not an FP at PL 1
+##! cron
+cron@
 crontab
+cron~
```

Comment:
`cron` isn't causing false positives but your also disabling them?

Reply:
`cron` isn't, which is why I've commented it, but `crontab` is, and so there might be others with the `cron` prefix that also do.